### PR TITLE
Add a color fill classmethod to Texture

### PR DESCRIPTION
**Pull request by pushfoo**
## tl;dr
1. Added `arcade.Texture.create_filled`, which is like `create_empty` but takes a color
2. Made `arcade.SpriteSolidColor` use it
3. Tested affected examples and building the doc, all looks ok

### 1. Easier prototyping for end users
Some users may begin their project by displaying game state through`Sprite.color` changes applied to solid white `SpriteCircle` or `SpriteSolidColor` instances. However, it can be annoying to rewrite such code to use texture data.

Merging this PR will make it easier for users to generate filler textures from the start. Switching to loading texture data from files is easy, regardless of whether they use [`load_textures`](https://api.arcade.academy/en/latest/api/texture.html#arcade-load-textures), [`load_spritesheet`](https://api.arcade.academy/en/latest/api/texture.html#arcade-load-spritesheet), or another method of their choice.

The code for projects that expand on #1204 could also be cleaner as a result. For example, some RTS games use a convention of red for low health, yellow for medium health, and green for full health. Users will no longer need to understand Pillow's API or image channels to achieve this.

### 2. Make writing examples easier
The piano example for #1198 will be easier to write cleanly with this merged.

### 3. Avoid memory leaks
It's unclear whether this comment is still relevant, but [`Sprite.color` may create memory leaks](https://github.com/pythonarcade/arcade/pull/974#issuecomment-931342070):
> [pvcraven](https://github.com/pvcraven) commented [on Sep 30, 2021](https://github.com/pythonarcade/arcade/pull/974#issuecomment-931342070)
My initial, quick-look-through, makes me think this is a good change.
>
>If someone color-cycles we'll have a memory leak, but that will mess up the atlas anyways. Makes me wonder if there should be docs somewhere that explain to use .color or otherwise explain the internals a bit.

### CODE REVIEW

**Comment by pushfoo** [comment]
A few questions for reviewers:
1. Should I add a series creation method that returns `List[Texture]` from a parameter sequence?
2. Is the current method name ok?
3. Should `color` have a default value?
